hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/security/x509/certificate/client/CertificateClientTestImpl.java

```diff
@@ -95,6 +97,7 @@ public CertificateClientTestImpl(OzoneConfiguration conf, boolean autoRenew)
       throws Exception {
     certificateMap = new ConcurrentHashMap<>();
     securityConfig = new SecurityConfig(conf);
+    rootCerts = ConcurrentHashMap.newKeySet();
```

Comment:
Why we need a set backed by a concurrent hashmap here? Isn't the simple or worst case the concurrent HashSet work?